.github/workflows/pytest.yaml

```diff
@@ -27,6 +28,18 @@ jobs:
           - "sqlite,rest-sqlite,postgres,rest-postgres"
         include:
           # with pyarrow
+          - python-version: "3.13"
+            with-pyarrow: true
+            postgres-version: "16"
+            pandas-version: false
+            backend: "sqlite,rest-sqlite"
+          # pgsql 15
+          - python-version: "3.13"
+            with-pyarrow: false
+            postgres-version: "15"
+            pandas-version: false
+            backend: "postgres,rest-postgres"
+          # with pyarrow
```

Comment:
I would say you can remove the python 3.12 postgres tests (Lines 49-53), if there is a specific issue with that combination we can turn them back on.